app/assets/stylesheets/bars.scss

```diff
@@ -1,3 +1,23 @@
 // Place all the styles related to the Bars controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.jumbotron{
```

Comment:
I'm not sure if we should overwrite bootstrap's classes

Reply:
☝️ good point. Consider creating new class.